scripts/benchmark/checks/deploy_dos.py

```diff
@@ -53,10 +53,13 @@
 
 W3_TX_DEFAULTS = {'gas': 0, 'gasPrice': 0}
 
+NUM_BLOCKS = 20
+NUM_TX = 10
```

Comment:
Heh, so these ones *are* default values, and I think the `DEFAULT_` prefix makes them clearer.
```suggestion
DEFAULT_NUM_TX = 10
```